scripts/utils.ts

```diff
@@ -46,13 +64,25 @@ export async function compileContracts(): Promise<CompilerOutput> {
 	return compilerOutput
 }
 
+export class ScriptError extends Error {
```

Comment:
Because we aren't using error codes with `$?` anymore - I would argue that this abstraction isn't needed at all.